moz_ingest/io_modules/decoders/moz_ingest/common.lua

```diff
@@ -268,7 +260,7 @@ function transform_message(hsr, msg)
         end
     end
 
-    if maxminddb then
+    if maxminddb and not msg.Fields.geoCountry then
```

Comment:
There is actually a preexisting bug (line 310).  If we are reprocessing records that have already been geocoded and the subdecoder fails the error messages will have two copies of the geo information (from the user perspective it will be invisible as they just fetch the first)